### DISCUSSION TITLE
infrastructure/synology/enable-ssh-root-login/

### DISCUSSION THREAD

**Discussion by giscus** [fire1ce/3os.org#114] [General]
# infrastructure/synology/enable-ssh-root-login/

Enable SSH Root Login on Synology NAS directly from the terminal. No need to log in as a user and then enter 

https://3os.org/infrastructure/synology/enable-ssh-root-login/

**Comment by heinzelio** [comment]
very helpful, thank you. 
after problems with activating ssh over the web ui from synology, your solution solved my problem. after editing the file with root privileges, it was possible to set the checkbox active in the system settings for Terminal and SNP on my synology. (syno DS918+ with dsm 7.1.1-42962)

cheers heinz

**Comment by heinzelio** [comment]
*note*
It wasn't possible for me to access the synology via ssh at all; I first had to adjust the sshd.config via telnet.
